src/plugins/data_source_management/public/plugin.ts

```diff
@@ -41,7 +41,7 @@ export interface DataSourceManagementPluginStart {
   getAuthenticationMethodRegistry: () => IAuthenticationMethodRegistry;
 }
 
-export const DSM_APP_ID = 'dataSourceManagement';
+export const DSM_APP_ID = 'dataSources';
```

Comment:
Looks like we are changing the global constant which is used in multiple places, can we tests if all the places using the constant are working as expected?

Reply:
yes, it's also used in dropdown header, and no data source component. Added video to the PR description -> testing section